packages/forms/src/elements/file-list/components/Close.tsx

```diff
@@ -21,7 +21,14 @@ const CloseComponent = React.forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivEl
     );
 
     return (
-      <StyledFileClose ref={ref} {...props} onMouseDown={onMouseDown}>
+      <StyledFileClose
+        ref={ref}
+        aria-label="Press delete to remove"
```

Comment:
also `useText`? We should give consumers a nudge to localize.

Reply:
Totally fair. I overlooked the localization part. Thanks!

Reply:
Fixed: https://github.com/zendeskgarden/react-components/pull/1509/commits/54c788cd319976a36af608e55fa1ee55adb3e4bc